packages/components/src/Actions/Action/Action.component.js

```diff
@@ -88,6 +89,10 @@ function Action(props) {
 		model,
 	});
 
+	if (form) {
+		buttonProps.form = form;
+	}
```

Comment:
buttonProps.form = form
if form is undefined will not adding anything. So for sure it's even better.